modules/installation-and-upgrade/pages/container-deployment/prepare-micro-host.adoc

```diff
@@ -0,0 +1,115 @@
+== Prepare {sle-micro} {microversion} Host
+
+.Procedure: Downloading the Installation Media
+. Locate the {sle-micro} {microversion} installation media at https://www.suse.com/download/sle-micro/.
+. Download [filename]``SLE-Micro-5.5-DVD-x86_64-GM-Media1.iso``.
```

Comment:
This needs to be changed to 6.1. We don't need to give exact but rather say that download the appropriate media  or around those lines.